libs/ImpSpec/ImpSpec.cabal

```diff
@@ -47,7 +47,8 @@ library
         prettyprinter-ansi-terminal,
         random,
         text,
-        unliftio
+        unliftio,
+        microlens
```

Comment:
Please revert any changes to `ImpSpec`. It has already been extracted into [its own repo](https://github.com/input-output-hk/ImpSpec) and will be removed from `cardano-ledger` very soon.

Reply:
See #4763